lib/brains/forms.py

```diff
@@ -97,16 +98,30 @@ def clean_plan(self):
         self.seller_product = obj
         return data
 
+    def format_descriptor(self, name):
+        # The rules for descriptor are:
+        #
+        # Company name/DBA section must be either 3, 7 or 12 characters and
+        # the product descriptor can be up to 18, 14, or 9 characters
+        # respectively (with an * in between for a total descriptor
+        # iname of 22 characters)
+        return 'Mozilla*{}'.format(name.replace(' ', ''))[0:22]
```

Comment:
it might be nice to log this so we can check back to see if something is nonsensical